### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by acosmin**
- The `display` argument will apply `esc_html` on the value, so it shouldn't be escaped. 
- `show-avatars` is a static string, it shouldn't be escaped.

### CODE REVIEW

File: template-parts/post/author-bio.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
  */
 
 ?>
-<div class="author-bio <?php echo get_option( 'show_avatars' ) ? esc_attr( 'show-avatars' ) : ''; ?>">
+<div class="author-bio <?php echo get_option( 'show_avatars' ) ? 'show-avatars' : ''; ?>">
```
Comment (carolinan) [inline template-parts/post/author-bio.php:11]: Good catch, mind fixing https://github.com/WordPress/twentytwentyone/blob/trunk/comments.php#L25 too?

Comment (acosmin) [reply template-parts/post/author-bio.php:11]: Sure, I'll do it first thing tomorrow :) mobile right now

Comment (acosmin) [reply template-parts/post/author-bio.php:11]: Done

**Comment by acosmin** [comment]
@carolinan there are also these two lines :) I think they need to be updated: https://github.com/WordPress/twentytwentyone/blob/9c22fe37620c7bbb064b7336f2a1d5c94c34ed6e/classes/class-twenty-twenty-one-customize-color-control.php#L6-L7

**Pull request merged**